docs/getting-started/create-configuration.md

```diff
@@ -716,7 +705,7 @@ rm -rf crossplane-config
 
```

Comment:
Maybe we can have an ending here with a link to `Concepts`, guiding people who want to understand everything in detail after finishing the flow here.

Reply:
Great idea!